unittests/rpc/handlers/AccountChannelsTest.cpp

```diff
@@ -55,7 +55,7 @@ TEST_F(RPCAccountHandlerTest, NonHexLedgerHash)
                 "ledger_hash": "xxx"
             }})",
             ACCOUNT));
-        auto const output = handler.process(input, yield);
+        auto const output = handler.process(input, Context{&yield});
```

Comment:
i'd prefer std::cref(yield) here if you can make it work with the idea i described above